blocks/math.js

```diff
@@ -82,12 +82,19 @@ Blockly.Blocks['math_positive_number'] = {
    * @this Blockly.Block
    */
   init: function() {
-    this.setHelpUrl(Blockly.Msg.MATH_NUMBER_HELPURL);
-    this.setColour(Blockly.Blocks.math.HUE);
-    this.appendDummyInput()
-        .appendField(new Blockly.FieldNumber('0', 0, Infinity), 'NUM');
-    this.setOutput(true, 'Number');
-    this.setOutputShape(Blockly.OUTPUT_SHAPE_ROUND);
+    this.jsonInit({
+      "message0": "%1",
+      "args0": [
+        {
+          "type": "field_number",
+          "name": "NUM",
+          "min": 0
```

Comment:
I think this is intentional. In my view a "positive number" is a "float > 0": use case is "wait _ seconds" (doesn't make sense to wait for -1 second, but does make sense to wait for 0.5 seconds). In my view a "whole number" is "unsigned int" or "int > 0": use case is "repeat _ times" where it doesn't make sense to repeat -1 times or .5 times. We could possibly have a "signed int" math block, but I haven't found a block that needs that yet. Do you have any in mind?